src/Login/style.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  width: 700px;
```

Comment:
hoe about mobile? will I see horizontal scroll?

Reply:
i cahange it to more suitable